content/warnings/refs-must-have-owner.md

```diff
@@ -4,45 +4,49 @@ layout: single
 permalink: warnings/refs-must-have-owner.html
 ---
 
-You are probably here because you got one of the following error messages:
+ أنت على الأرجح هُنا بسبب رسائل الخطأ التالية:
 
-*React 16.0.0+*
-> Warning:
+*React 16.0.0 و أعلى*
```

Comment:
There is no need to translate this line ( `+` ) ..

Reply:
yeah but it won't hurt 😅i think translating it would be a better fit since the second option (lower than 16) is translated ...